### PR TITLE
Fix some differences between GLES2 and GLES3 code.

### PR DESCRIPTION
**Pull request by fzurita**
I found a few other differences between GLSLCombiner_gles2.cpp/GLSLCombiner_ogl3x.cpp and Shaders_gles2.h/Shaders_ogl3x.h. Also, I did a search for #ifndef GLES2 and found one thing that didn't seem right in FrameBuffer.cpp so I fixed that too.

### CODE REVIEW

**Comment by loganmc10** [comment]
Sometimes they are different for a reason though. I don't think uDepthScale is used in GLES2 at all (it is used with fragment depth writing), so updating it in GLES2 is pointless

**Comment by fzurita** [comment]
Yep, I understand that. That one slipped by me.

**Comment by fzurita** [comment]
Ok, I fixed the issue you saw.

File: src/FrameBuffer.cpp [hunk 1]
```diff
@@ -788,7 +788,6 @@ void FrameBuffer_Destroy()
 	RDRAMtoColorBuffer::get().destroy();
 #ifndef GLES2
 	DepthBufferToRDRAM::get().destroy();
-	ColorBufferToRDRAM::get().destroy();
```
Comment (fzurita) [inline src/FrameBuffer.cpp]: ok, fixed.

**Comment by gonetz** [comment]
Merged: 16262e52ad4

**Pull request closed without merging**